opentelemetry-instrumentation-tower/development/config/grafana-datasource-mimir.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: 1
```

Comment:
can you add a readme.md file to the development folder explaining how to leverage the docker-compose, and the grafana dashboards?

I am also not sure how much detailed example we want to maintain per instrumentation. it may makes sense to move some of these to common examples, as they are applicable to any web apps...